src/Controllers/MyRadio/default.php

```diff
@@ -32,7 +32,8 @@
         ->addVariable('studio_demoed', $user->isStudioDemoed())
         ->addVariable('is_trainer', $user->isTrainer())
         ->addVariable('has_show', $user->hasShow())
-        ->addVariable('paid', $user->isCurrentlyPaid());
+        ->addVariable('paid', $user->isCurrentlyPaid())
+        ->addVariable('contract_signed', $user->hasSignedContract());
```

Comment:
Starting to think we should just expose $user->toDataSource generally... but that's a seperate issue